codexctl/device.py

```
@@ -1,1024 +1,1124 @@
+            paramiko.client.SSHClient: SSH client object for the device.
+        """
+
+        if remote_address is None:
+            remote_address = self.get_remarkable_address()
+            self.address = remote_address  # For future reference
+        else:
+            if self.check_is_address_reachable(remote_address) is False:
+                raise SystemError(f"Error: Device {remote_address} is not reachable!")
+
+        client = paramiko.client.SSHClient()
+        client.set_missing_host_key_policy(paramiko.AutoAddPolicy())
```

Ruff (0.15.15)

[error] Paramiko call with policy set to automatically trust the unknown host key

(S507)